requirements.txt

```diff
@@ -39,3 +39,5 @@ rich
 beautifulsoup4
 premailer
 html-diff
+pytesseract
```

Comment:
New requirements must be added to all platform specific requirement files as well - `requirements-win.txt`, `requirements-mac.txt`, `requirements-linux.txt`